packages/rancher-cis-benchmark/charts/values.yaml

```diff
@@ -5,7 +5,7 @@
 image:
   cisoperator:
     repository: rancher/cis-operator
-    tag: v1.0.5-rc2
+    tag: v1.0.5
   securityScan:
     repository: rancher/security-scan
```

Comment:
Yes, @cbron this is an issue with the way the index.yaml gets generated that needs to be tackled in https://github.com/rancher/charts-build-scripts/issues/54.

The TLDR for why this does not get modified is because the chart has already been added to the index.yaml... Helm only generates the digest on the first time a chart is added to an index, it never modifies existing charts tracked in the index.yaml.

But you are right that *as of now* we do not need it. If we eventually get to signing Helm charts, this will be necessary in the future.